docs/features/custom-errors.md

```diff
@@ -0,0 +1,65 @@
+# Custom Errors
+
+## Introduction
+
+This package allows you to return [JSON API error objects](http://jsonapi.org/format/1.0/#error-objects) which can include various information about what went wrong.
+
+## Custom Errors In Controllers
+
+This is particularly useful for errors that you have to manually present to the user through the controller. In order to magically return errors, you would need to use the `ErrorsAwareTrait`.
+
+
+Below is an example of a scenario which a user fails to login due to incorrect credentials.
+
+```php
+namespace App\Http\Controllers\Api\V1;
+
+use CloudCreativity\JsonApi\Contracts\Http\Requests\RequestInterface as JsonApiRequest;
+use CloudCreativity\JsonApi\Document\Error;
+use CloudCreativity\JsonApi\Utils\ErrorsAwareTrait;
+
+class JsonWebTokensController extends Controller
+{
+    use ErrorsAwareTrait;
```

Comment:
You don't need to use this trait in the a controller - the trait is intended for validators.